src/httpClient.ts

```diff
@@ -171,6 +171,25 @@ export default class httpClient {
         return filtered_records;
     }
 
+    async getAirtableRecordsForCountry(filters: StudiesFilters) {
+
+        const reqBody: Record<string, any> = {
+            filters: filters,
+            include_subgeography_estimates: true,
+        }
+
+        const response = await this.httpPost('/data_provider/records', reqBody)
+        if (!response) {
+            return [];
+        }
+
+        const filtered_records = response.map((item: Record<string, any>) => {
```

Comment:
nit: we could use `forEach` here instead of `map` since we just want to mutate the original array right?